distributed/deploy/cluster.py

```diff
@@ -70,8 +89,22 @@ async def _start(self):
         self._watch_worker_status_task = asyncio.ensure_future(
             self._watch_worker_status(comm)
         )
+
+        with contextlib.suppress(KeyError):
+            self.cluster_info.update(
+                (await self.scheduler_comm.get_metadata(keys=["cluster-manager-info"]))
+            )
+        self._sync_cluster_info_task = asyncio.ensure_future(self._sync_cluster_info())
         self.status = Status.running
 
+    async def _sync_cluster_info(self):
+        while True:
+            await self.scheduler_comm.set_metadata(
+                keys=["cluster-manager-info"],
+                value=copy.copy(self.cluster_info),
```

Comment:
is this copy necessary?

Reply:
Yeah in `LocalCluster` there ends up being one dict with a pointer in the cluster and scheduler. I spent a while trying to figure out why it was synced before I had even added the sync code.

It's not going to be a pointer for implementations other than `LocalCluster` so adding `copy.copy` ensures consistent behaviour.